stable/rabbitmq-ha/templates/service.yaml

```diff
@@ -40,17 +40,23 @@ spec:
     - name: http
       protocol: TCP
       port: {{ .Values.rabbitmqManagerPort }}
+      {{- if eq .Values.service.type "NodePort" }}
```

Comment:
`nodePort` is also applicable to service.type LoadBalancer (see https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.15/#serviceport-v1-core)

Reply:
Good point! Fixed that.